packages/components/src/components/status-badge/status-badge.scss

```diff
@@ -12,9 +12,23 @@
         align-items: center;
         white-space: nowrap;
 
+        &--pending {
+            font-size: var(--text-size-xxxs);
+            color: var(--status-warning);
+            background-color: var(--status-warning-transparent);
+            border: 1.5px;
```

Comment:
is it really expected?
I think all our deriv CSS values should be multiple of 8, so we may use less than 8, but it should be 4 or 2

Reply:
hey @yauheni-deriv is there any particular reason why we should stick to multiples of 8 or values less than 2

Reply:
As per design, it is 4 @Dhruv-deriv . could you please check

Reply:
@Dhruv-deriv , this is kind of logic and steps, which our designers follow

Reply:
its fixed